src/components/Header.jsx

```diff
@@ -0,0 +1,47 @@
+import { Link, NavLink } from "react-router-dom"
+import { useSelector } from 'react-redux'
+
+import '../styles/header.scss'
+
+const Header = ({ searchMovies }) => {
+  
+  const { starredMovies } = useSelector((state) => state.starred)
```

Comment:
please define selectors as separate function in separate folder, they can and should be reused